ArduPlane/quadplane.cpp

```diff
@@ -1563,7 +1563,7 @@ float QuadPlane::get_pilot_input_yaw_rate_cds(void) const
     bool manual_air_mode = plane.control_mode->is_vtol_man_throttle() && (air_mode == AirMode::ON);
     if (!manual_air_mode &&
         plane.get_throttle_input() <= 0 && !plane.control_mode->does_auto_throttle() &&
-        plane.arming.get_rudder_arming_type() != AP_Arming::RudderArming::IS_DISABLED) {
+        plane.arming.get_rudder_arming_type() != AP_Arming::RudderArming::IS_DISABLED && !(inertial_nav.get_velocity_z() < .5 * get_pilot_velocity_z_max_dn())) {
```

Comment:
```suggestion
        plane.arming.get_rudder_arming_type() != AP_Arming::RudderArming::IS_DISABLED && !(inertial_nav.get_velocity_z() <-0.5 * get_pilot_velocity_z_max_dn())) {
```
I think your right, get_velocity_z is NEU, so descending would be a negative number get_pilot_velocity_z_max_dn returns a positive number. 

I wonder if we should do a more gradual cutoff rather than a sharp change. Might be abit odd having it cut in and out. But if its fine in SITL....

Reply:
will change.....dont think gradual is needed since is to fix an issue only if stick is within DZ of full low....otherwise yaw already works...so only at maximum descent being commanded....so it should be fine I would think

Reply:
pushed a fix and restested